app/mobile-content/pages/edit-page-dialog/edit-page-dialog.component.html

```diff
@@ -1,16 +1,22 @@
-<h1 md-dialog-title>{{ 'create exhibit page' | translate }}</h1>
+<h1 md-dialog-title>
+  {{
+    page.id === -1 ?
```

Comment:
You actually can't use an `if` statement here, or indeed any kind of statement, because Angular's interpolation mechanism requires an expression. Put an `if` there and watch Angular spit an interpolation error at you.